<?xml version="1.0" encoding="UTF-8"?>
<testsuite hostname="Hexagon" name="com.psx.technology.debug.c3po.content.parser.XDebugParserTest" tests="6" failures="5" timestamp="30 Jul 2011 17:50:41 GMT" time="9.065" errors="0">
  <testcase name="handleLineDataEntry" time="0.015" classname="com.psx.technology.debug.c3po.content.parser.XDebugParserTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;initForm&amp;gt; but was:&amp;lt;_initForm&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<initForm> but was:<_initForm>
	at com.psx.technology.debug.c3po.content.parser.XDebugParserTest.handleLineDataEntry(XDebugParserTest.java:29)
... Removed 30 stack frames]]>
    </failure>
  </testcase>
  <testcase name="handleLineDataExit" time="0.0010" classname="com.psx.technology.debug.c3po.content.parser.XDebugParserTest">
    <skipped/>
  </testcase>
  <testcase name="splitMethodNameWithClass" time="0.0010" classname="com.psx.technology.debug.c3po.content.parser.XDebugParserTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;TestClass&amp;gt; but was:&amp;lt;null&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<TestClass> but was:<null>
	at com.psx.technology.debug.c3po.content.parser.XDebugParserTest.splitMethodNameWithClass(XDebugParserTest.java:72)
... Removed 30 stack frames]]>
    </failure>
  </testcase>
  <testcase name="splitMethodNameWithClassAndNamespace" time="0.0" classname="com.psx.technology.debug.c3po.content.parser.XDebugParserTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;TestClass&amp;gt; but was:&amp;lt;null&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<TestClass> but was:<null>
	at com.psx.technology.debug.c3po.content.parser.XDebugParserTest.splitMethodNameWithClassAndNamespace(XDebugParserTest.java:80)
... Removed 30 stack frames]]>
    </failure>
  </testcase>
  <testcase name="parseFileWithoutReplace" time="4.777" classname="com.psx.technology.debug.c3po.content.parser.XDebugParserTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;33232&amp;gt; but was:&amp;lt;58670&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<33232> but was:<58670>
	at com.psx.technology.debug.c3po.content.parser.XDebugParserTest.parseFileWithoutReplace(XDebugParserTest.java:100)
... Removed 30 stack frames]]>
    </failure>
  </testcase>
  <testcase name="parseFileWithReplace" time="4.246" classname="com.psx.technology.debug.c3po.content.parser.XDebugParserTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;33232&amp;gt; but was:&amp;lt;58670&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<33232> but was:<58670>
	at com.psx.technology.debug.c3po.content.parser.XDebugParserTest.parseFileWithReplace(XDebugParserTest.java:90)
... Removed 30 stack frames]]>
    </failure>
  </testcase>
</testsuite>
